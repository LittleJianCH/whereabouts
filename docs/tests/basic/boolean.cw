query (x, y) {
  equation x = true
  equation y = false
}
