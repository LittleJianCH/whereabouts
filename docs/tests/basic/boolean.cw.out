{ "solutions": [{ "x": true, "y": false }] }
